src/StatsTools/reweighting.jl

```diff
@@ -394,18 +405,241 @@ function mixed_estimator_df_progress(shift::Vector, hproj::Vector, vproj::Vector
     return DataFrame(nts)
 end
 
+"""
+    rayleigh_replica_estimator(
+        op_ol, vec_ol, shift, h, dτ;
+        skip = 0,
+        E_r = mean(shift[skip+1:end]),
+        weights = w_exp,
+        kwargs...,
+    ) -> r::RatioBlockingResult
+    rayleigh_replica_estimator(
+        df::DataFrame;
+        shift="shift",
+        op_ol="Op1", 
+        vec_ol="dot", 
```

Comment:
```suggestion
        shift_name="shift",
        op_name="Op1", 
        vec_name="dot", 
```
I'd prefer to rename the keyword arguments to clarify that they are names rather than the vectors `op_ol` etc., as the original naming might cause confusion with the "argument" `op_ol` and the "keyword" argument `op_ol` having quite different meaning despite having the same name.

A further question: Why do we need keyword arguments to pass the strings `"dot"` and `"shift"'? What is the use case for not using the default values?

Reply:
Yes that makes sense for the arguments. I was just following the convention from the other functions `growth_estimator` and `mixed_estimator`. Shall I change those as well? and merge the doc strings for the vector and DataFrame versions?

As for the default values, an obvious use case is when processing data that has used importance sampling where the relevant vector overlap data is not called "dot", as determined by `AllOverlaps`.

Reply:
Yes to changing the docstrings for `mixed_estimator` and `growth_estimator` and merging  docs for vector and  df version.

> As for the default values, an obvious use case is when processing data that has used importance sampling where the relevant vector overlap data is not called "dot", as determined by AllOverlaps.

Ah, OK.